libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -309,12 +309,24 @@ void AP_Follow::handle_msg(const mavlink_message_t &msg)
             _target_location.set_alt_cm(packet.alt / 10, Location::AltFrame::ABSOLUTE);
         }
 
+        // get a local timestamp with correction for transport jitter
+        uint32_t timestamp_ms = _jitter.correct_offboard_timestamp_msec(packet.time_boot_ms, AP_HAL::millis());
+        uint32_t dt = (timestamp_ms - _last_location_update_ms) * 0.001f;
```

Comment:
This should be a float? Maybe that is why your getting 0 dt?
```suggestion
        const float dt = (timestamp_ms - _last_location_update_ms) * 0.001f;
```

Reply:
There's also the case that `_last_location_update_ms` is not initialized in the header file. So, in the first iteration we're really just getting garbage out of it. Wasn't sure if it was OK to set at as 0 though.